Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/Batching/Batching.xaml.cs

```diff
@@ -1,263 +1,265 @@
-using Microsoft.AI.MachineLearning;
```

Comment:
Are these just spacing changes?

Reply:
that was a mistake, reset all changes to the file